npm/angular/src/mount.ts

```diff
@@ -0,0 +1,225 @@
+import 'zone.js'
```

Comment:
Might want to move side-effects into a separate file, though this would increase the complexity of scaffolding during launchpad setup.

Reply:
I think this illustrates the problem of trying to have no side effects - this import makes perfect sense here, it's required for Angular, right?

An alternative would be scaffold it in `component.ts`, but then we increase the complexity of `component.js` for... not much benefit. imo. Also, is there a downside to having people import/bundle `zone.js` in E2E? (If they don't want to, they just shouldn't mix their E2E and CT support code).

Reply:
There shouldn't be any issues for importing `zone.js` in e2e tests. I was harping on not having side effects a while ago but now I would rather have side-effects if it means for a cleaner support file and less complexity during project setup.